infoschema/infoschema.go

```diff
@@ -402,6 +402,18 @@ func (is *infoSchema) RuleBundles() []*placement.Bundle {
 	return bundles
 }
 
+func (is *infoSchema) SetPolicy(policy *model.PolicyInfo) {
```

Comment:
I think it's better to make SetPolicy/DeletePolicy private or move them to builder too.

Reply:
make sense